components/TextControls/BUILD

```diff
@@ -27,45 +27,80 @@ load(
 
 licenses(["notice"])
 
+mdc_extension_objc_library(
+    name = "Shared",
```

Comment:
this is just the enums? should we call it "enums"

Reply:
hmm looks like more than enums.
This is kind of a wild card target as is. Can we start off making it visibility = private for now?

Reply:
I thought it was just two enums. Am I missing something? I can make it visibility private

Reply:
I think the risk is future maintainers might start "dumping" things in there. It seems trivial now, but it could easily start to creep into a "wild card" target over time.

Reply:
sorry i got confused and found the private/shared ones and thought they were in this target.

Let us call this "enums"

Reply:
Done